docs/usage.md

```diff
@@ -954,7 +954,9 @@ aws s3 --no-sign-request ls s3://annotation-cache/vep_cache/
 
 Since both Snpeff and VEP are internally figuring the path towards the specific cache version / species, `annotation-cache` is using an extra set of keys to specify the species and genome build.
 
-So if you are using this resource, please either use the `--use_annotation_cache_keys`, or point towards the specific species, genome and build matches the directory structure within the cache.
+So if you are using this resource, please either set `--use_annotation_cache` to use the AWS annotation cache, or point towards your own cache folder structure matching the expected structure.
```

Comment:
```suggestion
So if you are using this resource, please either set `--use_annotation_cache_keys` to use the AWS annotation cache, or point towards your own cache folder structure matching the expected structure.
```

Reply:
I actually find it without "Keys" more obvious, but maybe too big of a change at this point